### PR TITLE
📝 docs: update Android development environment hints

### PR DESCRIPTION
**Pull request by jyyi1**
We are missing some important pieces regarding the manual Android development environment setup. In this change set, I updated the documents with the latest Android Studio (version 2020.3.1).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +53,7 @@ If this gives you unexpected Cordova errors, run:
 
     npm run clean && npm ci && npm run action gulp -- build android
 
-Cordova will generate a new Android project in the platforms/android directory.  Install the built apk by  platforms/android/build/outputs/apk/android-armv7-debug.apk
+Cordova will generate a new Android project in the platforms/android directory. Install the built apk by `platforms/android/app/build/outputs/apk/<processor>/debug/app-<processor>-debug.apk`
```
Comment (daniellacosse) [inline README.md:56]: it would be nice to have a way to easily detect the correct processor, or just mention where to find it

**Review by daniellacosse — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -34,8 +34,10 @@ UI components are located in [src/www/ui_components](src/www/ui_components). The
 
 Additional requirements for Android:
 
-* Android Studio 4+
-* Android SDK 29
+* [Android Studio 2020.3.1+](https://developer.android.com/studio)
+* [Latest Android Sdk Commandline Tools](https://developer.android.com/studio/command-line)
+* Android SDK 30 (with build-tools) via commandline `sdkmanager "platforms;android-30" "build-tools;30.0.3"`
```
Comment (fortuna) [inline README.md:39]: This is very helpful. But don't we need the NDK as specified at https://github.com/Jigsaw-Code/outline-client/blob/master/tools/build/setup_linux_android.sh?

File: README.md [hunk 3]
```diff
@@ -34,8 +34,10 @@ UI components are located in [src/www/ui_components](src/www/ui_components). The
 
 Additional requirements for Android:
```
Comment (fortuna) [inline README.md:35]: No to address in this PR, but we have this Docker image that was intended to make build easier, but it's probably broken at this point: https://github.com/Jigsaw-Code/outline-client/blob/master/tools/build/Dockerfile

File: README.md (hunk 2 above)
Comment (daniellacosse) [reply README.md:39]: if they're starting from scratch, they'll need to add these as well - https://nisrulz.com/android-tips-tricks/android-aliases.html

Comment (jyyi1) [reply README.md:39]: @fortuna From my experience I did not install NDK and build still succeeds.

@daniellacosse The environment variables (e.g., `ANDROID_HOME`) mentioned in the link are deprecated, I only need to add the following two:

```
export ANDROID_SDK_ROOT=$HOME/Android/Sdk
export PATH="$ANDROID_SDK_ROOT/cmdline-tools/latest/bin:$PATH"
```

File: README.md (hunk 3 above)
Comment (jyyi1) [reply README.md:35]: Will try later.

File: README.md (hunk 1 above)
Comment (jyyi1) [reply README.md]: added a hint, but will let the developer to figure it out.

**Review by fortuna — APPROVED** [review]

**Pull request merged**